etc/profile-m-z/mimetype.profile

```diff
@@ -0,0 +1,67 @@
+# Firejail profile for mimetype
+# Description: Determines the file type
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include mimetype.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
```

Comment:
```suggestion
#blacklist ${RUNUSER}
```

[As mentioned][1], the program appears to be intended to be used on arbitrary
files on the system and other programs may depend on it.

Also, since it uses `quiet`, this may lead to silent breakage of unrelated
programs that call it.

[1]: https://github.com/netblue30/firejail/pull/6247#pullrequestreview-1912649150

Reply:
I've implemented your suggestions, but added `blacklist ${RUNUSER}/wayland-*`.